server/src/main/java/org/opensearch/action/admin/cluster/node/stats/RestActionsStats.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.action.admin.cluster.node.stats;
+
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.common.io.stream.Writeable;
+import org.opensearch.common.xcontent.ToXContentFragment;
+import org.opensearch.common.xcontent.XContentBuilder;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Rest actions responses stats
+ *
+ * @opensearch.internal
+ */
+public class RestActionsStats implements Writeable, ToXContentFragment {
+    private Map<String, Map<Integer, Integer>> restActionsStatusCount;
```

Comment:
Can we use Long here?

Reply:
I think status count won't go beyond INT range, so making it LONG, will cause memory wastage.